content/docs/quickstart/_index.md

```diff
@@ -116,7 +116,7 @@ Please make sure you have `kubectl` installed (see [kubernetes.io/docs/tasks/too
     keptn version
     ```
 
-#### 3. Install Keptn and authenticate Keptn CLI
+#### 2.2 Install Keptn and authenticate Keptn CLI
 
```

Comment:
If we change the headers, we will break all existing links to this page. I advise against changing this to 2.2

Reply:
Valid point - changed it back.